.evergreen/config/functions.yml

```diff
@@ -72,15 +72,10 @@ functions:
   "prepare resources":
     - command: shell.exec
       params:
+        working_dir: src
         script: |
           ${PREPARE_SHELL}
-          rm -rf $DRIVERS_TOOLS
-          if [ "${project}" = "drivers-tools" ]; then
-            # If this was a patch build, doing a fresh clone would not actually test the patch
-            cp -R ${PROJECT_DIRECTORY}/ $DRIVERS_TOOLS
-          else
-            git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
-          fi
+          git submodule update --init --depth 1
```

Comment:
The `--depth 1` was not in PHPLIB. I can add it also to reduce the Git history that get downloaded.

Reply:
I have to revert this change, there is a git checkout inside the libmongoc submodule when `LIBMONGOC_VERSION` is set (always). Maybe I can fetch the tag directly.

https://github.com/mongodb/mongo-php-driver/blob/bf34d5bd895d289206d70a725cb61420c665bb2e/.evergreen/config/functions.yml#L139-L145

Reply:
I suppose you could initialize the submodules independently by passing their path as an argument to `git submodule update`, although that would make this more fragile if additional submodules are introduced to the project.

Assuming it works, fetching the tag for libmongoc does seem more resilient, and then you could keep that logic local to the `LIBMONGOC_VERSION` handling.